pkgs/development/libraries/boost/cross-compile-175.patch

```diff
@@ -0,0 +1,7 @@
+--- a/bootstrap.sh
++++ b/bootstrap.sh
+@@ -225,4 +225,4 @@
+   pwd=`pwd`
+-  (cd "$my_dir/tools/build/src/engine" && ./build.sh)
++  (cd "$my_dir/tools/build/src/engine" && BUILD_CXX="$CXX_FOR_BUILD" ./build.sh cross-cxx)
```

Comment:
same question here.

Reply:
This case is different as the value of TOOLSET is not passed anymore since https://github.com/boostorg/boost/commit/d3170a7d517f7ca0646ad7aeff64b0579409af47 . That basically broke the standard path for cross compilation.

cc @pdimov

Reply:
cc @grafikrobot

Reply:
@grafikrobot,

In your change to line 226 https://github.com/boostorg/boost/commit/d3170a7d517f7ca0646ad7aeff64b0579409af47#diff-99d5a2a8ca4290b61836b3ad37941f2b925e9ff3151c90dcd5ea82364e4122a1L226
you stop passing "$TOOLSET" to the ./build.sh, is this intentional?